README.rst

```diff
@@ -4,42 +4,46 @@ Kubernetes on AWS
 
 **WORK IN PROGRESS**
 
-This repo contains some scripts and templates to provision Kubernetes_ clusters on AWS using Cloud Formation and CoreOS_.
+This repo contains configuration templates to provision Kubernetes_ clusters on AWS using Cloud Formation and CoreOS_.
 
-**Consider this as very early alpha quality**. Many values are hardcoded, and currently we're focusing on solving our own, specific/Zalando user case. However, **we are open to ideas from the community at large about potentially turning this idea into a project that provides universal/general value to others**. Please contact us via our Issues Tracker with your thoughts and suggestions.
+**Consider this as alpha quality**. Many values are hardcoded, and currently we're focusing on solving our own, specific/Zalando user case.
+However, **we are open to ideas from the community at large about potentially turning this idea into a project that provides universal/general value to others**.
+Please contact us via our Issues Tracker with your thoughts and suggestions.
 
-It was initially based on `kube-aws`_, but we decided to diverge from it:
+Configuration in this repository initially was based on kube-aws_, but now depends on three components which are currently not open sourced yet:
```

Comment:
The last sentence sounds a bit weird to me. I would say:

*but now depends on three components which are not yet open sourced*: